udata_front/frontend/__init__.py

```diff
@@ -77,3 +77,10 @@ def test_500():
     if app.config.get('DEBUG_TOOLBAR'):
         from flask_debugtoolbar import DebugToolbarExtension
         DebugToolbarExtension(app)
+
+    # Security override init
+    from udata.auth import init_security
+    from udata_front.forms import ExtendedAgainRegisterForm
```

Comment:
`ExtendedRegisterForm` (as a name) should be enough now that we have this one in udata-front?